NGOSignUp/signUpNGO.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<head>
+	<link rel="stylesheet" type="text/css" href="styleSignUp.css">
+	<script src="jquery-2.1.3.min.js" type="text/javascript">
+
+
+
+
+
+
+
+
+
+	</script>
+</head>
+<div class="container">
+<form id="signup" enctype="application/json" method="post" name="signupform">
+	<div class="header">
+		<h3>SIGNUP</h3>
+	</div>
+	<div class="seperator"></div>
+	<div class="inputs">
+
+		<input type="text" name="name" placeholder="Name of the organisation*" autocomplete="on" autofocus required/><br>
+		<input type="email" placeholder="Email*" required/><br>
+		<input type="password" placeholder="Password*" required/><br>
+		<input type="text" maxlength="10" placeholder="Phone no*" required/><br>
+		<input type="text" maxlength="10" placeholder="Secondary Phone no"/><br>
+		<p>Address</p>
+		<input type="text" placeholder="Street Address 1*" required/><br>
+		<input type="text" placeholder="Street Address 2"/><br>
+		<input type="text" placeholder="City*" required/><br>
+		<input type="text" placeholder="State*" required/><br>
+		<input type="text" placeholder="Country*" required/><br>
+		<input type="text" placeholder="PIN"/>
```

Comment:
if you are adding a pin then a validation must be performed based on country if it is a valid pin